### PR TITLE
Add marketing text

### PR DESCRIPTION
**Pull request by lgillie**
Add marketing text

### CODE REVIEW

File: src/OpenstoreSchema/Core/Entity/ProductTranslation.php [hunk 1]
```diff
@@ -95,6 +95,11 @@ class ProductTranslation
      */
     private $usp;
 
+    /**
+     * @ORM\Column(type="string", length=2000, nullable=true, options={"comment" = "Product marketing text"})
```
Comment (belgattitude) [inline src/OpenstoreSchema/Core/Entity/ProductTranslation.php]: @lgillie Good, let me check few things about length... Mysql have some limitations regarding how many chars, indexes... are allowed for a single table. Travis will help us too.

Comment (belgattitude) [reply src/OpenstoreSchema/Core/Entity/ProductTranslation.php]: Ok all good for the length checks... Travis have a problem right now installing Mariadb 10.3... that's why it fails... We can ignore.

https://github.com/belgattitude/openstore-schema-core/commit/9bd0989e04badee4078ce4231e351d8e3e48782c

File: src/OpenstoreSchema/Core/Entity/ProductTranslation.php [hunk 2]
```diff
@@ -95,6 +95,11 @@ class ProductTranslation
...
+     */
+    private $marketing_text;
```
Comment (belgattitude) [inline src/OpenstoreSchema/Core/Entity/ProductTranslation.php]: @lgillie 

IMO `marketing_text` is not an ideal name... Description is already taken... Do you have an alternative name that does no convey the *department* originating request.  Is it a `short_description` ? That could help to stay generic... Any idea or marketing_text is good for you ?

**Comment by belgattitude** [comment]
@lgillie 

thanks for P/R... before I merge could you take a look at my reviews ? 

As a side note.... `usp` or `marketing_text` as much as they make sense for the people using those fields may not be widely used in a web context... For clarity I would always consider a different naming for the DB field, letting the user call it whatever feels good for him.. Not a general rule, just a thought to keep in mind if they start adding more and more 'in-house' terms... 

Second... If you add columns into 'ProductDescription'... you make it hard to link a text valid for multiple products later... No problem neither... But product_serie_translation can be good place too. 

Just to consider... For now all is good

File: src/OpenstoreSchema/Core/Entity/ProductTranslation.php (hunk 2 above)
Comment (belgattitude) [reply src/OpenstoreSchema/Core/Entity/ProductTranslation.php]: 'short_description' can be recyclable in other contexts too ;)  What you think ?

**Comment by lgillie** [comment]
Hello,
I have changed the name of the new field. I think it's more global now.
Thank you

**Comment by belgattitude** [comment]
Merged. Thanks

**Pull request merged**